File: Payhub - SDK/c#_SDK/PayHubSDK/PayHubSDK/com/payhub/ws/model/YearlySchedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace PayHubSDK.com.payhub.ws.model
{
    [DataContract]
    public class YearlySchedule
    {
        [DataMember]
        private String year_to_start;
        public String Year_to_start
        {
            get { return this.year_to_start; }
            set { if (value != null)this.year_to_start = value; }
        }
        [DataMember]
        private int? yearly_bill_on_month_number;
        public int? Yearly_bill_on_month_number
        {
            get { return this.yearly_bill_on_month_number; }
            set { if (value != 0)this.yearly_bill_on_month_number = value; }
        }
        [DataMember]
        private String yearly_bill_on_day_of_month;
        public String Yearly_bill_on_day_of_month
        {
            get { return this.yearly_bill_on_day_of_month; }
            set { if (value != null)this.yearly_bill_on_day_of_month = value; }
        }

    }
}

File: Payhub - SDK/java_SDK/PayHubSDK/src/main/java/com/payhub/ws/model/YearlySchedule.java
/**
 * 
 */
package com.payhub.ws.model;


import com.fasterxml.jackson.annotation.JsonInclude;

/**
 * @author agustin
 *
 */
@JsonInclude(JsonInclude.Include.NON_NULL)  
public class YearlySchedule {
	private String year_to_start;
	private Integer yearly_bill_on_month_number;
	private String yearly_bill_on_day_of_month;

	public String getYear_to_start() {
		return year_to_start;
	}

	public void setYear_to_start(String year_to_start) {
		this.year_to_start = year_to_start;
	}

	public Integer getYearly_bill_on_month_number() {
		return yearly_bill_on_month_number;
	}

	public void setYearly_bill_on_month_number(
			Integer yearly_bill_on_month_number) {
		this.yearly_bill_on_month_number = yearly_bill_on_month_number;
	}

	public String getYearly_bill_on_day_of_month() {
		return yearly_bill_on_day_of_month;
	}

	public void setYearly_bill_on_day_of_month(
			String yearly_bill_on_day_of_month) {
		this.yearly_bill_on_day_of_month = yearly_bill_on_day_of_month;
	}

	  
}
